src/Microsoft.DotNet.ProjectModel/Graph/LibraryDependencyType.cs

```diff
@@ -5,45 +5,24 @@
 
 namespace Microsoft.DotNet.ProjectModel.Graph
 {
-    public struct LibraryDependencyType
+    public struct LibraryDependencyType : IEquatable<LibraryDependencyType>, IEquatable<string>
     {
-        public static LibraryDependencyType Default = LibraryDependencyType.Parse("default");
+        public static LibraryDependencyType Empty = new LibraryDependencyType();
+        public static LibraryDependencyType Default = new LibraryDependencyType("default");
+        public static LibraryDependencyType Build = new LibraryDependencyType("build");
+        public static LibraryDependencyType Platform = new LibraryDependencyType("platform");
 
-        public static LibraryDependencyType Build = LibraryDependencyType.Parse("build");
+        public string Value { get; }
 
-        public LibraryDependencyTypeFlag Flags { get; private set; }
```

Comment:
We weren't using most of the flags, so @davidfowl and I discussed it and decided to just use the keywords instead.